app/views/content/train-to-be-a-teacher/if-you-have-a-degree.md

```diff
@@ -1,71 +1,89 @@
+heading: "Train to be a teacher if you have a degree"
 description: |-
   Discover how you can train to be a teacher and gain qualified teacher status (QTS) if you have a degree. Including school-led and university-led training.
 related_content:
-    Get support training to teach if you're disabled: "/get-support-training-to-teach-if-you-are-disabled"
+    Get support training to teach if you're disabled : "/get-support-training-to-teach-if-you-are-disabled"
+    Get school experience : "/train-to-be-a-teacher/get-school-experience"
     Improve your subject knowledge : "/train-to-be-a-teacher/subject-knowledge-enhancement"
     Who do you want to teach? : "/train-to-be-a-teacher/who-do-you-want-to-teach"
-    Initial teacher training : "/train-to-be-a-teacher/initial-teacher-training"
 promo_content:
     - content/train-to-be-a-teacher/promos/find-your-course
     - content/train-to-be-a-teacher/promos/events-near-you
 navigation: 20.05
+navigation_title: If you have a degree
 navigation_description: Find out how to get QTS if you have a degree or you’re studying for one now. There are different options you can choose.
+keywords:
+  - becoming a teacher
+  - teacher training
+  - school direct
+  - pgce
+  - pgde
 ---
 
-If you have a degree, or you’re currently studying for one, there are different routes you can take to getting [qualified teacher status](/what-is-qts) (QTS). You need QTS to work in the majority of schools in England including state maintained primary, secondary and special schools.
+You need [qualified teacher status (QTS)](/what-is-qts) to teach in most primary, secondary and special schools in England. Once you have a degree you can get QTS through [teacher training](/train-to-be-a-teacher/initial-teacher-training). 
 
-We also have information for [international students who want to apply for teacher training](/non-uk-teachers/train-to-teach-in-england-as-an-international-student).
+Postgraduate teacher training courses usually take 9 months full time, in line with the academic year, or 18 to 24 months part time. 
 
-## Study for a PGCE or PGDE
-You can study for [a PGCE or PGDE](/what-is-a-pgce) (an academic qualification) with QTS at a university. Most of the time you will be in schools on placement but there is some academic learning. Courses typically take one year but you may be able to study part-time.
+If you’re not from the UK, find out about [training to teach in England](/non-uk-teachers/train-to-teach-in-england-as-an-international-student).
 
-Fees for full-time courses are usually around £9,250 but you can:
+## Types of teacher training
 
-- get [funding that you do not have to pay back](/funding-your-training#bursaries-and-scholarships) for certain subjects
-- [apply for a student loan](/funding-your-training#tuition-fee-and-maintenance-loans)
+Your teacher training course might be led by a university, school, college or group of schools. There are different teacher training types, including:
```

Comment:
```suggestion
Your teacher training course might be led by a university, school, college or group of schools. There are different teacher training types, including:
```

Rather than saying there are different types, maybe it is more like 'You may/will see these referred to as....' (reworded!)